test/e2e-upgrade-tests.sh

```diff
@@ -45,9 +45,11 @@ function install_latest_release() {
     yaml="serving-core.yaml"
   fi
 
-  install_knative_serving \
-    "${url}/${yaml}" \
-    || fail_test "Knative latest release installation failed"
+  wget "${url}/${yaml}" -O "${yaml}" \
```

Comment:
What's the idea behind the extra download step? Debugging?

Reply:
`ko` doesn't support urls, but has better `--selector` support than `kubectl`